techniques/fileDistribution/copyGitFile/2.1/metadata.xml

```diff
@@ -212,9 +226,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         </CONSTRAINT>
       </INPUT>
 
-    </SECTION>
+  </SECTION>
 
-    <SECTION name="Post-modification hook" component="true" multivalued="false" componentKey="COPYFILE_NAME" displayPriority="low">
+    <SECTION name="WINDOWS specific options" multivalued="false" component="false" displayPriority="low">
```

Comment:
why is it empty ??

Reply:
No windows options for the moment, maybe in future versions.